servers/v3-proxy-api/src/graph/toGraphQL.spec.ts

```diff
@@ -3,25 +3,125 @@ import { setSaveInputsFromGetCall } from './toGraphQL';
 import {
   SavedItemsSortBy,
   SavedItemsSortOrder,
-  SavedItemStatusFilter,
 } from '../generated/graphql/types';
+import { defaultQuery } from '../test/fixtures';
 
 describe('toGraphQL', () => {
-  //todo: refactor this test as you refactor the method
   it('should map saves input', () => {
     const expected = {
       pagination: {
         limit: 30,
         offset: 0,
       },
       sort: {
-        sortBy: SavedItemsSortBy.UpdatedAt,
-        sortOrder: SavedItemsSortOrder.Asc,
-      },
-      filter: {
-        status: SavedItemStatusFilter.Unread,
+        sortBy: SavedItemsSortBy.CreatedAt,
+        sortOrder: SavedItemsSortOrder.Desc,
       },
+      filter: {},
     };
-    expect(setSaveInputsFromGetCall('sample-rest-params')).toEqual(expected);
+    expect(setSaveInputsFromGetCall(defaultQuery)).toEqual(expected);
+  });
+  describe('filter object', () => {
+    // Need to see how Express handles this
+    it.todo('handles duplicate query parameters');
```

Comment:
todo ticket?

Reply:
I actually just took care of it -- it combines them into an array. Added a test case (caught by the validator -- this todo was before I had done the validator)